src/ir/enum_ty.rs

```diff
@@ -71,16 +71,20 @@ impl Enum {
         };
 
         declaration.visit(|cursor| {
-            if cursor.kind() == CXCursor_EnumConstantDecl {
```

Comment:
Makes sense.  Here's a new version with the outer `if`, but still without `unwrap()`, for the reason above.  If I understand rust correctly, there shouldn't be any runtime time penalty, right?